support/Python/Schedule.py

```diff
@@ -349,7 +351,10 @@ def schedule(
     context.update(executable_name=executable.name)
 
     # Resolve CLI
-    spectre_cli = shutil.which("spectre")
+    # This is the path of the `spectre` CLI where this script is installed.
+    # We may have to make this more robust for resubmissions if we run into
+    # problems or unexpected behavior.
+    spectre_cli = Path(__file__).parent.parent.parent.parent / "spectre"
```

Comment:
This isn't as fragile as you think because this is in the build directory (not in `SPECTRE_HOME`). So it only breaks if we move the location of the python package inside the build dir.

This method ensures that we use the same CLI for resubmissions. The previous method would use whichever CLI is in the `PATH`, which we can't control.

Reply:
Added a test for this too

Reply:
This will work even in `PY_DEV_MODE`?

Reply:
Yep, symlinks aren't resolved